### ISSUE TITLE
Trying to get Codewars Red

### ISSUE THREAD

**Issue by MarkSchemmer** [Codewars/codewars.com#1317]
I keep giving my card's credentials, for payment on www.codewars.com, I keep receiving a response of "Subscription failure: Your card number is incorrect." When I use this card all the time and I know it works (I even checked with my bank).
I would like to buy Codewars Red to contribute to the website because, I learn a lot, I receive help, and finally I would like codewars.com to generate more revenue so the website can continue to grow and develop.
I can pay with my Visa debit card, Paypal, or Bitcoin.
I'm sorry to say I don't feel comfortable paying with Alipay.

**Comment by MarkSchemmer** [comment]
It would be nice if you could add PayPal

**Comment by jhoffner** [comment]
This is unfortunate. We definitely want to add PayPal support, however Stripe is our current provider and there is no PayPal option. Adding PayPal would require us to support two billing systems. 

Switching over to using https://www.patreon.com/ might be an option. I'll look into it.

**Comment by tachyonlabs** [comment]
As above, I tried again today, on both Firefox and Chrome, with Discover and Amex cards in addition to Visa and MasterCard, with no change. :-(

**Comment by njohnson7** [comment]
If anyone else is still experiencing this issue (@tachyonlabs), I finally found a card that works: https://www.netspend.com/

It's a pretty terrible card for everything else, but it's the only one I've tried that works for Codewars for some reason

**Comment by tachyonlabs** [comment]
Yesterday evening I thought I would try again to see if things were working yet -- I tried with one of my Visa cards, and again got the message that it was declined. Then I had the thought that I should go check my card transactions online to make sure that Stripe wasn't billing my cards even though it was saying they were declined. I checked my Discover card, and saw that the web site showed a Stripe payment and deposit for $1 each that were still listed as "Pending" despite having dates of June 29. I then tried paying for Codewars Red with that Discover card again, but was once again declined. I don't know what the deal is, Stripe is certainly a legit company ...
